.changeset/rude-mangos-marry.md

```diff
@@ -0,0 +1,8 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer': patch
+'@spreadshirt/backstage-plugin-s3-viewer-backend': patch
+'@spreadshirt/backstage-plugin-s3-viewer-common': patch
+'@spreadshirt/backstage-plugin-s3-viewer-node': patch
+---
+
+Bump Backstage dependencies to version 1.31.1
```

Comment:
In #119 we said that the old backend system will be removed, is that not happening yet?

Reply:
I could do it as well. Backstage still has them in the code, but they seem to be currently deprecating all the occurrences first.

I can imagine 2 options here: completely remove it now and add a breaking change; or leaving it for one more release and then remove all completely

Reply:
Ok, then let's leave it for the next release.  Could you do it as a separate MR without the upgrade to the next version though?  That wouldn't need a release, but it would separate the breaking changes and the actual version upgrade.